### ISSUE TITLE
wiiupro show nothing on the controller tab.

### ISSUE THREAD

**Issue by AnarkoDX** [KeyPuncher/WiinUPro#119]
hello again XP , im trying with wiinupro , and the controller seems to connect , but the controller tab of the program doesnt show anything .thanks in advance. 
![wiinupro](https://user-images.githubusercontent.com/75274357/222978392-58b14075-6a7a-4279-a1f8-1d64773767bf.jpg)

**Comment by AnarkoDX** [comment]
Update , i loaded a profile that i found here in github , the controller tab still doesnt show  anything but in windows devices seems to be detecting the keystrokes. so it works in a way at least.

**Comment by KeyPuncher** [comment]
I believe this is what usually occurs when it is unable to start reading from the controller. Usually a failure for Windows to provide the data stream. I am assuming that you've connected the controller via Windows Bluetooth.

Hmmm.. If the inputs are working just nothing to display then must be a something else. Try double clicking around in the blank space to see if it opens something, it seems unlikely but perhaps the images failed to load.

**Comment by AnarkoDX** [comment]
yeah i tried clicking everywhere but doesnt open anything. my windows is screwed in some weird way.
